app/models/lookup_value.rb

```diff
@@ -24,6 +24,10 @@ class LookupValue < ActiveRecord::Base
   scoped_search :on => :match, :complete_value => true
   scoped_search :in => :lookup_key, :on => :key, :rename => :lookup_key, :complete_value => true
 
+  def value_present?
+    !!value.nil?
```

Comment:
also added a test which I should have added before, and would have caught this silly error of mine :smile: